localstripe/resources.py

```diff
@@ -244,6 +244,67 @@ def do_expand(path, obj):
         return obj
 
 
+class Balance(object):
+    object = 'balance'
+
+    def __init__(self, **kwargs):
+        if kwargs:
+            raise UserError(400, 'Unexpected ' + ', '.join(kwargs.keys()))
+
+        self.object = self.object
+        self.livemode = False
+        self.type = 'balance'
+        self.connect_reserved = []
```

Comment:
If it's not implemented, the best is probably not to include `connect_reserved`. Would you be OK to remove it?

Reply:
Yes.